### PR TITLE
7387 Document searches are coming soon!

### PR DESCRIPTION
**Pull request by ericsorenson**
image

### CODE REVIEW

File: web-api/src/app-public.js [hunk 1]
```diff
@@ -33,24 +33,29 @@ const {
...
 const { getPublicJudgesLambda } = require('./public-api/getPublicJudgesLambda');
 const { todaysOpinionsLambda } = require('./public-api/todaysOpinionsLambda');
 
+// Temporarily disabled for story 7387
+// const {
+//   opinionPublicSearchLambda,
+// } = require('./public-api/opinionPublicSearchLambda');
+// const {
+//   orderPublicSearchLambda,
+// } = require('./public-api/orderPublicSearchLambda');
```
Comment (matthopson) [inline web-api/src/app-public.js:47]: Wouldn't you still be able to hit the lambda directly? Or did we intend to only terminate the api endpoint?

**Comment by matthopson** [comment]
Seems like we've got a lot of tests to disable.

File: web-api/src/app-public.js (hunk 1 above)
Comment (ericsorenson) [reply web-api/src/app-public.js:47]: Isn't this the only way to hit the useCase?

Comment (matthopson) [reply web-api/src/app-public.js:47]: ~Fairly sure you could call that lambda directly. Like from CLI~ I see what you're saying now!

**Review by rachelschneiderman — APPROVED** [review]

**Review by matthopson — APPROVED** [review]
🎉

**Pull request merged**